torch/csrc/jit/init.cpp

```diff
@@ -194,6 +195,11 @@ void initJITBindings(PyObject* module) {
           "_jit_pass_quant_fusion",
           [](std::shared_ptr<Graph>& g) { return QuantFusion(g); })
       .def("_jit_pass_fold_convbn", &FoldConvBatchNorm2d)
+      .def("_freeze_module",
+          [](script::Module& module) {
+            return freezeModule(module);
+          },
+          py::arg("module"))
```

Comment:
nit: `.def("_jit_pass_freeze_module", &freezeModule)`
also `freezeModule` --> `FreezeModule` for consistent naming?

Reply:
freeze_module is not a pass. Once we are confident with freeze_module. There should be a follow up patch to  expose freeze_module API as public.

I agree I will  make it consistent  :)